lib/sshutils/sftp/parse.go

```diff
@@ -39,13 +41,40 @@ type Destination struct {
 	Path string
 }
 
+var windowsDrivePattern = regexp.MustCompile(`^[A-Z]:\\`)
```

Comment:
Nit: I'm not sure if it's important in this context, but technically, Windows can accept paths that don't have a backslash after the colon. Path `C:foo\bar` would then be interpreted as "`foo\bar`, relative to the current directory on drive C".